src/sqlfluff/dialects/dialect_exasol.py

```diff
@@ -2563,6 +2464,29 @@ class UserLDAPAuthSegment(BaseSegment):
     )
 
 
+@exasol_dialect.segment(replace=True)
+class DropUserStatementSegment(BaseSegment):
+    """A `DROP USER` statement with CASCADE option.
+
+    https://docs.exasol.com/sql/drop_user.htm
+    """
+
+    type = "drop_user_statement"
+
+    is_ddl = False
+    is_dml = False
+    is_dql = False
+    is_dcl = True
+
+    match_grammar = Sequence(
+        "DROP",
+        "USER",
+        Ref("IfExistsGrammar", optional=True),
+        Ref("NakedIdentifierSegment"),
```

Comment:
Do you know if this can be a quoted identifier in Exasol?

In a lot of other dialects this is possible.
e.g. Postgres:
![image](https://user-images.githubusercontent.com/80432516/150606474-7280abb5-4bdf-428c-b61b-0ef34d746213.png)

Same question applies to all the `DROP` segments with `NakedIdentifierSegment` in this PR?

(also I appreciate you didn't introduce them in this PR, but good opportunity to check since I'm not an exasol user 😄 )

Reply:
Thanks for the hint. Its may bad pratice to use double quotes for identifier (my personal meaning) but its possible. I will look into this and fix the impacted locations. 

We should make this in another PR because there are more than the `DROP` statements impacted (e.g. create user/role...)

Reply:
Yeah totally agree with separating the fix into a separate PR but just wanted to check 👍 

FWIW I'm also strongly against using quoted identifiers, but there are others less sane 😄